examples/multimodal/wds.py

```diff
@@ -1,42 +1,34 @@
 from datachain import C, DataChain
 from datachain.lib.webdataset import process_webdataset
 from datachain.lib.webdataset_laion import WDSLaion, process_laion_meta
+from datachain.sql.functions import path
 
-wds = (
+wds_images = (
     DataChain.from_storage("gs://datachain-demo/datacomp-small/shards")
-    .filter(C("file.name").glob("00000000.tar"))
+    .filter(C("file.name").glob("000000[0-5]*.tar"))  # from *00.tar to *59.tar
     .settings(cache=True)
     .gen(laion=process_webdataset(spec=WDSLaion), params="file")
-    .save()  # materialize chain to avoid downloading data multiple times
```

Comment:
FYI this save should be fine if you want to keep it since it doesn't write embeddings.